packages/components/src/confirm-dialog/stories/index.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import React, { useState } from 'react';
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import { Heading } from '../../heading';
+import { ConfirmDialog } from '..';
+
+export default {
+	component: ConfirmDialog,
+	title: 'Components (Experimental)/ConfirmDialog',
+	parameters: {
+		knobs: { disabled: false },
```

Comment:
We should move away from `knobs` and rewrite this story to use `controls` instead — perfect task for a follow-up PR !

Reply:
Sounds good! Note taken!